e2e/test/scenarios/custom-column/cc-help-text.cy.spec.js

```diff
@@ -40,7 +40,7 @@ describe("scenarios > question > custom column > help text", () => {
 
     // Click outside of formula field instead of blur
     // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Expression").click();
+    cy.findByText("Expression").parent().click();
```

Comment:
Clicking the "Expressions" label now focuses the input, so we now click its parent element to lose focus